recipes/caf/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "0.17.6":
```

Comment:
Do you need 0.17.6? Otherwise, I would say to remove it.

Reply:
Yes - 0.18.0 is a major breaking change and people will be on 0.17.6 for a while.  It's the last of the compatible releases.